modules/standard/Random.chpl

```diff
@@ -482,6 +540,27 @@ module Random {
      See http://www.pcg-random.org/
      and the paper, "PCG: A Family of Simple Fast Space-Efficient Statistically
      Good Algorithms for Random Number Generation" by M.E. O'Neill.
+
+     Besides a Chapel :class:`RandomStream` interface, this module includes a
+     number of low-level PCG random functions:
+
+       * :record:`pcg_setseq_64_xsh_rr_32_rng` (the default PCG RNG)
+       * :record:`pcg_setseq_64_rxs_m_xs_64_rng`
+       * :record:`pcg_setseq_32_rxs_m_xs_32_rng`
+       * :record:`pcg_setseq_16_rxs_m_xs_16_rng`
+       * :record:`pcg_setseq_8_rxs_m_xs_8_rng`
+       * :record:`pcg_setseq_N_rxs_m_xs_N_rng` which is a generalization of the
+         above
+
+     These names come from the PCG paper and reference implementations.  The
+     first integer is the number of state bits, and the last integer is the
+     number of output bits. The other parts describe the permutation function in
+     the LCG.
+
+     It also includes some Chapel-specific features, such as generating real,
+     imag, and complex numbers; and generating numbers in a range in parallel.
+     These features are not available in the reference implementations of PCG.
+
```

Comment:
Ultimately, would it be possible to push these into a helper module of some sort, if for no other reason than to make the PCGRandom documentation less overwhelming (and overwhelmed by low-level routines?).  This feels like one of those Chapelerific- vs. system- interfaces that Kyle was characterizing in his packaging deep-dive.